.env.local.example

```
@@ -15,3 +15,10 @@ TYPESENSE_ADMIN_KEY=[your-admin-key]
 TYPESENSE_SEARCH_KEY=[your-search-key]
 TYPESENSE_SYNC_KEY=[your-sync-key]
 TYPESENSE_COLLECTION=objects
+
+NOTION_API_KEY=secret_xxxxxxxx
+NOTION_DATA_SOURCE_ID=[your-notion-data-source-id]
+NOTION_WEBHOOK_VERIFICATION_TOKEN=secret_xxxxxxxx
+NOTION_SYNC_APP_URL=https://[your-app-origin]
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NOTION_SYNC_APP_URL key should go before the NOTION_WEBHOOK_VERIFICATION_TOKEN key

(UnorderedKey)